sushy/interop_profiles/ServiceRoot.v1_0_0.json

```diff
@@ -0,0 +1,73 @@
+{
+	"SchemaDefinition": "RedfishInteroperabilityProfile.v1_0_0",
+	"ProfileName": "RedfishProfile",
+	"ProfileVersion": "1.0.0",
+	"Purpose": "Redfish Interoperability profile",
+	"OwningEntity": "Dell",
+	"ContactInfo": "abc@dell.com",
+	"Resources": {
+		"ServiceRoot": {
+			"PropertyRequirements": {
+				"Chassis": {
+				        "Comparison":"LinkToResource",
+                                        "Values":["ChassisCollection"]
+				},
+				"Managers": {
```

Comment:
As with `Chassis` above, I believe this property's `ReadRequirement` is `Recommended` and therefore must be explicitly provided.